public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -191,6 +212,8 @@ export const QueryAndExpressionsStep: FC<Props> = ({ editingExistingRule, onData
                   {...field}
                   dataSourceName={dataSourceName}
                   showPreviewAlertsButton={!isRecordingRuleType}
+                  lazyDefaultQuery={lazyDefaultQueries?.[0]}
```

Comment:
How do we know that we always want to pass the first query?

Reply:
the first query is the one that contains the data query (not expressions) as we do in `getDefaultQueriesAsync`